Detectors/GlobalTracking/CMakeLists.txt

```diff
@@ -1,65 +1,67 @@
-# Copyright 2019-2020 CERN and copyright holders of ALICE O2.
-# See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
-# All rights not expressly granted are reserved.
+#Copyright 2019 - 2020 CERN and copyright holders of ALICE O2.
```

Comment:
Could you restore the formatting of CMakeLists?

Reply:
it is changed now